enatega-multivendor-app/src/screens/Otp/Phone/PhoneOtp.js

```diff
@@ -118,21 +131,16 @@ function PhoneOtp(props) {
                 onPress={() => resendOtp()}>
                 <TextDefault
                   H4
-                  textColor={currentTheme.buttonTextPink}
+                  textColor={currentTheme.fontfourthColor}
                   style={alignment.MLsmall}
                   bold>
                   {t('resendBtn')}
                 </TextDefault>
               </TouchableOpacity>
             )}
           </View>
-          <View style={alignment.MBxSmall}>
```

Comment:
What's the purpose of removing this, is it done intentionally or by mistake, please confirm.

Reply:
No need of taking a separate <view> to give Margin bottom, instead I have given margin bottom to main <view>